### ISSUE TITLE
Adding a join table in Rails -- right direction?

### ISSUE THREAD

**Issue by lisawilliams** [ga-wdi-boston/full-stack-project#909]
Hi! 
My app reminds users of chores.  I have a resource for users, and a resource for chores. 

I would like users to be able to do basic CRUD actions on chores (create chore, show chore or list of chores, update a chore, delete a chore).  

So I *think* I want to create an association like this: Users have_many chores; chores belong_to users.  And I *think* I need to create a join table between chores and users.  

This is what my models and schema look like now, and below them is the `generate scaffold` command I think I should run. Am I going in the right direction? I'm a little hesitant just to run generators and mess up my models :D 

My schema: 
```ActiveRecord::Schema.define(version: 20170608035932) do

  # These are extensions that must be enabled in order to support this database
  enable_extension "plpgsql"

  create_table "chores", force: :cascade do |t|
    t.string   "chore_name"
    t.string   "directions"
    t.string   "day_of_week"
    t.datetime "created_at",  null: false
    t.datetime "updated_at",  null: false
  end

  create_table "examples", force: :cascade do |t|
    t.text     "text",       null: false
    t.integer  "user_id",    null: false
    t.datetime "created_at", null: false
    t.datetime "updated_at", null: false
    t.index ["user_id"], name: "index_examples_on_user_id", using: :btree
  end

  create_table "kids", force: :cascade do |t|
    t.string   "given_name"
    t.string   "family_name"
    t.datetime "created_at",  null: false
    t.datetime "updated_at",  null: false
  end

  create_table "users", force: :cascade do |t|
    t.string   "email",           null: false
    t.string   "token",           null: false
    t.string   "password_digest", null: false
    t.datetime "created_at",      null: false
    t.datetime "updated_at",      null: false
    t.index ["email"], name: "index_users_on_email", unique: true, using: :btree
    t.index ["token"], name: "index_users_on_token", unique: true, using: :btree
  end

  add_foreign_key "examples", "users"
end
```

models/user.rb: 

```class User < ApplicationRecord
  include Authentication
  has_many :chores
end
```

models/chore.rb:

```class Chore < ApplicationRecord
  belongs_to :user
end
```

And here's the `generate scaffold` command I think I should run to create a join table between these two resources: 

```rails generate model UsersHaveChores chores:references user:references```

Questions: Am I going in the right direction? Second, should I wait to put the belongs_to and has_many code in the models?  Sanity check appreciated :D

**Comment by jordanallain** [comment]
why do you think you need a join table for this? join tables are for many-to-many relationships.

**Comment by lisawilliams** [comment]
Okay, so I just need a reference, like in the schema for examples?

**Comment by jordanallain** [comment]
if the relationship you have is one-to-many

**Comment by lisawilliams** [comment]
ok kind of like in our one-to-many lesson with authors:books 

`bin/rails generate migration AddAuthorToBooks author:references`

**Comment by jordanallain** [comment]
yep! just think about which table you want storing the reference

**Comment by lisawilliams** [comment]
I don't know, I am stuck. I'd like to get one CRUD action working before the weekend starts so I can actually work on the weekend.

**Comment by lisawilliams** [comment]
Okay, so I ran: 

`rails generate migration AddUserToChore user:belongs_to`

Which generated the following migration file: 

```class AddUserToChore < ActiveRecord::Migration[5.0]
  def change
    add_reference :chores, :user, foreign_key: true
  end
end
```
My table `chores` now has the following columns: 

` id | chore_name |   directions    | day_of_week |         created_at         |         updated_at         | user_id `

The user_id column is blank.  From what I remember from the [Rails API One-to-Many] lesson, we played around in `bin/rails console` to assign authors to books. I'm going to try that now.
